pkg/tests/cross-tests/adapt.go

```diff
@@ -102,17 +102,23 @@ func (ta *typeAdapter) NewValue(value any) tftypes.Value {
 		switch v := value.(type) {
 		case map[string]any:
 			values := map[string]tftypes.Value{}
-			for k, el := range v {
-				values[k] = fromType(aT[k]).NewValue(el)
+			for key, expectedType := range aT {
```

Comment:
Noted that tftypes.Value representation insists (via panics) that every attribute has an entry, even if it's a nil entry. It also insisted on no optional attributes. This is now the case in this adapter, it frees the test writer from writing explicit nulls for attributes that do not matter.